index.bs

```diff
@@ -252,21 +252,22 @@ A <dfn>lock request</dfn> represents a pending request for a [=lock-concept|lock
 
 A [=lock request=] is a tuple of (<dfn>agent</dfn>, <dfn>clientId</dfn>, <dfn>origin</dfn>, <dfn>name</dfn>, <dfn>mode</dfn>, <dfn>promise</dfn>).
 
-Each origin has an associated <dfn>lock request queue</dfn>, which is a [=queue=] of [=lock requests=].
+Each origin has an associated <dfn>lock request queue set</dfn>, which is a [=set=] of [=lock request queues=] (a [=lock request queue=] being a [=queue=] of [=lock requests=]) where each queue is associated with a particular resource name.
 
 A [=lock request=] |request| is said to be <dfn>grantable</dfn> if the following steps return true:
 
 1. Let |origin| be |request|'s [=lock request/origin=].
-1. Let |queue| be |origin|'s [=lock request queue=]
+1. Let |queue set| be |origin|'s [=lock request queue set=]
+1. Let |name| be |request|'s associated [=lock request/name=]
+1. Let |queue| be |name|'s associated [=lock request queue=] in |queue set|
 1. Let |held| be |origin|'s [=held lock set=]
 1. Let |mode| be |request|'s associated [=lock request/mode=]
-1. Let |name| be |request|'s associated [=lock request/name=]
 1. If |mode| is "{{exclusive}}", then return true if all of the following conditions are true, and false otherwise:
     * No [=lock-concept|lock=] in |held| has a [=lock-concept/name=] that equals |name|
-    * No [=lock request=] in |queue| earlier than |request| has a [=lock request/name=] that equals |name|.
```

Comment:
This gets simpler, yay!

Reply:
Sorry, I'm not sure I see how this gets much simpler?

Reply:
Apologies - as mentioned offline, I was just happy that some text was deleted from the spec. Less words = better!